views/petition/share.mote

```diff
@@ -4,18 +4,34 @@
 <link rel="stylesheet" type="text/css" href="/css/share.css"/>
 
 
-<!-- FB SHARE config -->
-<div id="fb-root"></div>
+<!-- FB INIT AND SHARE config -->
 <script>
-  (function(d, s, id) {
-    var js, fjs = d.getElementsByTagName(s)[0];
-    if (d.getElementById(id)) return;
-    js = d.createElement(s); js.id = id;
-    js.src = "//connect.facebook.net/en_GB/sdk.js#xfbml=1&version=v2.0";
-    fjs.parentNode.insertBefore(js, fjs);
-  }(document, 'script', 'facebook-jssdk'));
+  window.fbAsyncInit = function() {
+    FB.init({
+      appId      : '1623988591156709',
```

Comment:
We could set this id inside the .env file. It's going to be published anyways as it's inside a template, but it's for consistency, so we have all those id numbers in one file.
After adding that number in the .env file, it should be 'grabbed' in the app.rb file (as the other variables from the .env file) and make use of it in the templates as {{ FB_ID }} for example.